build/rocm/README.md

```diff
@@ -1,38 +1,209 @@
+
+Once installed, verify ROCm installation using:
 
 ```Bash
-docker run -it -d --network=host --device=/dev/kfd --device=/dev/dri --ipc=host --shm-size 64G --group-add video --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -v ./:/jax --name rocm_jax jax-rocm:latest /bin/bash
+> rocm-smi
+
+========================================== ROCm System Management Interface ==========================================
+==================================================== Concise Info ====================================================
+Device  [Model : Revision]    Temp        Power     Partitions      SCLK     MCLK     Fan  Perf  PwrCap  VRAM%  GPU%
+        Name (20 chars)       (Junction)  (Socket)  (Mem, Compute)
+======================================================================================================================
+0       [0x74a1 : 0x00]       50.0°C      170.0W    NPS1, SPX       131Mhz   900Mhz   0%   auto  750.0W    0%   0%
+        AMD Instinct MI300X
+1       [0x74a1 : 0x00]       51.0°C      176.0W    NPS1, SPX       132Mhz   900Mhz   0%   auto  750.0W    0%   0%
+        AMD Instinct MI300X
+2       [0x74a1 : 0x00]       50.0°C      177.0W    NPS1, SPX       132Mhz   900Mhz   0%   auto  750.0W    0%   0%
+        AMD Instinct MI300X
+3       [0x74a1 : 0x00]       53.0°C      176.0W    NPS1, SPX       132Mhz   900Mhz   0%   auto  750.0W    0%   0%
+        AMD Instinct MI300X
+======================================================================================================================
+================================================ End of ROCm SMI Log =================================================
 ```
 
-***
-### JAX ROCm Releases
-We aim to push all ROCm-related changes to the OpenXLA repository. However, there may be times when certain JAX/jaxlib updates for
-ROCm are not yet reflected in the upstream JAX repository. To address this, we maintain ROCm-specific JAX/jaxlib branches tied to JAX
-releases. These branches are available in the ROCm fork of JAX at https://github.com/ROCm/jax. Look for branches named in the format
-rocm-jaxlib-[jaxlib-version]. You can also find corresponding branches in https://github.com/ROCm/xla. For example, for JAX version
-0.4.33, the branch is named rocm-jaxlib-v0.4.33, which can be accessed at https://github.com/ROCm/jax/tree/rocm-jaxlib-v0.4.33.
+### Step 2: Install the Latest Version of JAX
 
-JAX source-code and related wheels for ROCm are available here
+Install the required version of JAX with ROCm support using pip:
 
 ```Bash
-https://github.com/ROCm/jax/releases
+> pip3 install jax[rocm]
 ```
 
-***Note:*** Some earlier jaxlib versions on ROCm were released on ***PyPi***. 
+### Step 3: Verify the Installed JAX Version
+
+Check whether the correct version of JAX and its ROCm plugins are installed:
+
+```Bash
+> pip3 freeze | grep jax
+jax==0.4.35
```

Comment:
Should we be referencing the latest 0.4.37 release here? Alternatively, we can drop the version number and just say that this should output some version of `jax`, `jax-rocm60-plugin` etc. Wdyt?

Reply:
@superbobry The main reason I included version numbers is that the ROCm JAX release lags behind the upstream JAX. For example, the latest ROCm JAX release is 0.4.35. If I omit the version details, it might install the latest versions of JAX and jaxlib (e.g., 0.4.37) alongside the current latest ROCm JAX release (0.4.35), leading to potential mismatches.